Bluebird2000/Questioner
# #163043759 General implementation of UI feedback

#### What does the PR do?

- Its the general implementation of user interface feedback

#### Description of Task to be completed?

- Add Signup page link on the index page
- Implement transitions on user login page using pure CSS
- Add Back link to login and signup page
-  Maintain consistency in post questions and comment ui page
- Add Date meetup was created to questioners page
-  Add a pointer cursor on upvote and downote link
- Refactor responsiveness on footer text not aligning
- Add create meetup link to Admin Dashboard 
- Implement UI (Admin Delete meetup page)
- clean up unnecessary line space in style.css
- Add a new line after end of file on user signup page

#### How should this be manually tested?
- Open index.html on the browser and ensure it loads correctly, click on Get started
- Open login.html on the browser and hover on the background to see the transition effect
- Open questioners.html on the browser and ensure the browser loads correctly
- Open questioners.html on the browser and hover the upvote and downvote link on questions
- Open admin/index.html and navigate to create-meetup link
- Open admin/delete-meetup.html to view the delete meetup page

#### Any background context you want to provide?
- This is a general implementation of feedback pointed out by Andela Bootcamp LFA Lanre

#### What are the relevant pivotal tracker stories?
[#163043759](https://www.pivotaltracker.com/story/show/163043759)

#### Screnshots
![get-started](https://user-images.githubusercontent.com/41438761/50952917-217bb380-14b2-11e9-920b-d8a2cc5146ca.PNG)

![transition-effect-login](https://user-images.githubusercontent.com/41438761/50953281-2f7e0400-14b3-11e9-828a-82ed9e881e28.PNG)

![upvote-downvote](https://user-images.githubusercontent.com/41438761/50953222-0493b000-14b3-11e9-81cc-d7c913b8fc11.PNG)
a8.PNG)

![delete-record](https://user-images.githubusercontent.com/41438761/50953237-12e1cc00-14b3-11e9-8074-2c30dddf59ff.PNG)
